libr/anal/fcn.c

```diff
@@ -48,35 +48,66 @@ R_API const char *r_anal_functiontype_tostring(int type) {
 }
 
 #if READ_AHEAD
-static R_TH_LOCAL ut64 cache_addr = UT64_MAX;
+	#if R2_580
+	// TODO: move into io :?
+	static int read_ahead(RAnal *anal, ut64 addr, ut8 *buf, int len) {
```

Comment:
do not indent functions or preprocessor lines, all of them should start on the first column